src/AureliaDependenciesPlugin.ts

```diff
@@ -13,7 +14,13 @@ class AureliaDependency extends IncludeDependency {
 
 class Template {
   apply(dep: AureliaDependency, source: Webpack.Source) {
-    source.replace(dep.range[0], dep.range[1] - 1, "'" + dep.request.replace(/^async(?:\?[^!]*)?!/, "") + "'");
+    // Get the module id, fallback to using the module request
+    let moduleId: string = dep.request;
+    if (dep.module && typeof dep.module[preserveModuleName] === 'string') {
+      moduleId = dep.module[preserveModuleName];
```

Comment:
I am not sure what the impact of this change is. 
Can you explain briefly, maybe with an example that behaves differently before/after this change?

Reply:
`AureliaDependenciesPlugin` uses the `preserveModuleName` symbol to set the generated moduleId. We then check that it is set here and use that for the webpack moduleId, otherwise fallback to the default usage.

Reply:
OK, I see. 
I had this idea of rewriting user code with Webpack generated id but it breaks quite a few things (such as conventions).

You take a middle ground where you rewrite user's code with our _normalized_ id. 
That seems safe and would solve a few issue, I _think_, as `aurelia-loader` will do the same normalization anyway.., and in the rare cases where it doesn't (there are a few) it would break at runtime anyway.

Looks like a good change to me 👍 

Out of curiosity: does this fix a specific problem you have?

Reply:
Yes it does see #121